packages/langium-sprotty/src/lsp.ts

```diff
@@ -0,0 +1,40 @@
+/******************************************************************************
+ * Copyright 2021 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
+
+import {
+    Connection, NotificationType
+} from 'vscode-languageserver';
+import { ActionMessage } from 'sprotty-protocol';
+import { OperationCancelled } from 'langium';
+import { LangiumSprottyServices } from './sprotty-services';
+
+/* eslint-disable @typescript-eslint/no-namespace */
+export namespace DiagramActionNotification {
```

Comment:
Side-note: I ask myself why we do lint namespaces away, they seem to be useful for us. Or is there a better alternative? Maybe we should just allow namespaces (part of another PR).